config/dispatcher/dispatcher.ex

```diff
@@ -297,6 +297,10 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://cache/publication-statuses/"
   end
 
+  match "/urgency-levels/*path", @any do
```

Comment:
Enkel de `get` HTTP method toelaten voor deze codelijst. Gebruikers kunnen concepten niet zelf bewerken, toevoegen of verwijderen.

Reply:
Okidoki, is aangepast